### PR TITLE
New option to spcify camera bearing

### PR DESCRIPTION
**Pull request by ToeBee**
This allows the handling of cameras that are reversed or off-center  from the direction of travel.

Note the check for < 0 but no check for > 360. Apparently exiv handles > 360 without complaining but it threw errors for any negative degrees.

### CODE REVIEW

**Comment by BathoryPeter** [comment]
Very useful option, I usually create +-90°images with my camera.
This script works fine, but writes original bearing to the output instead of corrected:

- print("Added geodata to: {}  time {}  lat {}  lon {}  alt {}  bearing {}".format(filename, time, lat, lon, elevation, bearing))
+ print("Added geodata to: {}  time {}  lat {}  lon {}  alt {}  bearing {}".format(filename, time, lat, lon, elevation, corrected_bearing))

**Comment by ToeBee** [comment]
Good catch. Fixed.

File: python/geotag_from_gpx.py [hunk 1]
```diff
@@ -78,8 +80,11 @@ def add_exif_using_timestamp(filename, time, points, offset_time=0):
         exiv_lat = (make_fraction(lat_deg[0],1), make_fraction(int(lat_deg[1]),1), make_fraction(int(lat_deg[2]*1000000),1000000))
         exiv_lon = (make_fraction(lon_deg[0],1), make_fraction(int(lon_deg[1]),1), make_fraction(int(lon_deg[2]*1000000),1000000))
 
+        corrected_bearing = bearing + offset_bearing
+        if corrected_bearing < 0:
```
Comment (ybkuang) [inline python/geotag_from_gpx.py]: We can actually do `corrected_bearing %= 360` here. It will handle both >360 and <0. Would you like to submit another pull request with the correction?

**Comment by ToeBee** [comment]
I have pushed an improvement to address your comment.

**Comment by ybkuang** [comment]
@ToeBee Great, thanks. It is now merged.
